CONTRIBUTING.md

```diff
@@ -68,6 +78,18 @@ scripts/format.sh
 - you must update the documentation
 - you must run the above scripts to format and line
 
+## Code development process
+
+1. Fork and clone repository
+2. Create a new virtual environment with conda/virtualenv
+3. Activate environment and install earthaccess as editable source
+    1. `pip install -e .`
+    2. `poetry install` (note, if we use Poetry with conda we need to tell poetry to not use its own virtual environments)
```

Comment:
I believe these are covered by the commands given earlier in the guide (not _exactly_ these commands, but same result).

Reply:
In my experience, I was unable to get these things working post `poetry install` from a fresh build. I consulted with @betolink and he provided the above instructions after I couldn't get the unit tests to run

Reply:
The steps at the top of the guide work for me, so what specific steps did you follow and what error did you encounter?

Reply:
I wonder if we can put these instructions on a separate "If you are not using Poetry, here is another way of getting a dev environment". Ideally, we should have a "universal" bash script ala `npm install dev` that will take care of everything including virtual environment and dev installs. In any case hopefully this PR gets merged soon-ish and we can keep iterating on it!